controllers/database_controller.go

```diff
@@ -501,6 +502,35 @@ func (r *DatabaseReconciler) createProxy(ctx context.Context, dbcr *kciv1alpha1.
 		}
 	}
 
+	crdList := crdv1.CustomResourceDefinitionList{}
```

Comment:
can you please add `if dbcr.IsMonitoringEnabled()`

Reply:
added over the proxyinterface.